packages/server-side-render/src/server-side-render.js

```diff
@@ -123,4 +107,40 @@ export class ServerSideRender extends Component {
 	}
 }
 
+ServerSideRender.defaultProps = {
```

Comment:
`defaultProps` is a feature of prop-types which we're not using in the Gutenberg project.  Instead, defaults should be set inline on the component.  You could probably define the defaults as their own functional components in this file and then just use them if the incoming render props are not set.

Reply:
@nerrad Can you check this again? I definitely found several files using `defaultProps`, and this code didn't break in testing?

Reply:
oh huh!  I learned something new I guess :). I always thought it was a feature available via prop-types (basically because of it's mention in [this doc](https://reactjs.org/docs/typechecking-with-proptypes.html)) but it makes sense it's a native feature to React.  Move along then, nothing to see here 😜